tests/schemas/pilot.schema.json

```diff
@@ -125,6 +132,75 @@
       "required": ["name", "text"],
       "additionalProperties": false
     },
+    "shipStats": {
```

Comment:
this is literally a direct copy/paste of the "stats" schema for ships. Makes me think there might be a better way to do this, but presumably any stat could be updated in the same way.

Reply:
I think it's an appropriate solution for now - if we change this in future we should change `shipActions` as well